src/main/resources/com/google/cloud/spanner/jdbc/DatabaseMetaData_GetTables.sql

```diff
@@ -15,16 +15,16 @@
  */
 
 SELECT TABLE_CATALOG AS TABLE_CAT, TABLE_SCHEMA AS TABLE_SCHEM, TABLE_NAME,
-       CASE WHEN TABLE_SCHEMA = '' THEN 'TABLE' ELSE 'VIEW' END AS TABLE_TYPE,
+       CASE WHEN TABLE_TYPE = 'BASE TABLE' THEN 'TABLE' ELSE TABLE_TYPE END AS TABLE_TYPE,
```

Comment:
This ensures that we continue to report what is called `BASE_TABLE` in Spanner as `TABLE` in JDBC to retain backwards compatibility. So this basically means:
- Return `'TABLE'` if Spanner calls it a `'BASE TABLE'`
- Return the `TABLE_TYPE` in all other cases

Reply:
When was this renamed?